app/javascript/widgets/landing_page/pages/home.jsx

```diff
@@ -62,20 +64,34 @@ const Home = () => {
 
         <Stack alignment="center">
           <div className="tw-text-xl tw-w-3/5 tw-mr-auto">
-            {"SAP's "} strategic Infrastructure-as-a-Service (IaaS) stack, optimised for SAP solutions, running purely
+            {"SAP's "} strategic Infrastructure-as-a-Service (IaaS) stack, optimized for SAP solutions, running purely
             in SAP datacenters.
           </div>
           <Stack direction="vertical" alignment="end" gap="1">
-            <Button
-              icon={selectedDomain ? "openInBrowser" : "place"}
-              variant="primary"
-              title={selectedDomain ? `Enter ${selectedDomain}` : "Select region/domain"}
-              className="whitespace-nowrap tw-py-1.5 tw-px-3"
-              onClick={handleHeroButtonClick}
-            >
-              {setHeroButtonText()}
-            </Button>
-            {selectedDomain && (
+            {hideDomainSwitcher === "true" || hideDomainSwitcher === true ? (
```

Comment:
You're ensuring that it's a boolean in `App.jsx`. So this can be simplified like this:
```suggestion
            {hideDomainSwitcher ? (
```